lib/apirequest.js

```diff
@@ -184,6 +186,10 @@ function createAPIRequest(parameters, callback) {
   delete options.auth; // is overridden by our auth code
   delete options.params; // We handle params ourselves and Request does not recognise 'params'
 
+  if (missingCallback) {
+    return options;
```

Comment:
Regarding this, I've found that most scenarios that require the `body` variable causes the request to be a multipart request. 
For example, `drive.insert({media: {body: readStream}, uploadType: 'multipart', resource: {title: 'Test'}})` will cause a multipart request to be formed in the current API. If I were to implement the ability to support this in batch requests, it would require a lot of string manipulation which I'm unsure is suitable for the quality of API, as I would have to generate the multipart string into the request manually. 
Supporting a request that has the `body` variable, while not being multipart is pretty easy to implement though. 
Should I attempt to implement support for multipart requests inside batch requests?